packages/contract/src/Account.sol

```diff
@@ -109,7 +117,10 @@ contract Account is BaseAccount, UUPSUpgradeable, Initializable {
             userOpHash
         ); // Emulate EIP-712 signing: https://eips.ethereum.org/EIPS/eip-712
         uint8 keySlot = uint8(userOp.signature[0]);
-        require(keys[keySlot][0] != bytes32(0) && keys[keySlot][1] != bytes32(0), "invalid key slot");
+        require(
+            keys[keySlot][0] != bytes32(0) && keys[keySlot][1] != bytes32(0),
+            "invalid key slot"
```

Comment:
nit but you could check just `keys[keySlot][0]` here & below

Reply:
done